src/main/plugin/iso19139.rndt/schematron/schematron-rules-rndt.sch

```diff
@@ -600,14 +600,13 @@ temporalSamplingService;temporalProximityAnalysisService;metadataProcessingServi
     <!--ESTENSIONE TEMPORALE-->
 	<sch:pattern>
 		<sch:title>$loc/strings/M200</sch:title>
-		<sch:rule context="//gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:extent/gmd:EX_Extent/gmd:temporalElement/gmd:EX_TemporalExtent/gmd:extent[//gml:TimePeriod]
-                                  |//gmd:MD_Metadata/gmd:identificationInfo/srv:SV_ServiceIdentification/srv:extent/gmd:EX_Extent/gmd:temporalElement/gmd:EX_TemporalExtent/gmd:extent[//gml:TimePeriod]">
+		<sch:rule context="//gmd:MD_Metadata/gmd:identificationInfo/*/*:extent/gmd:EX_Extent/gmd:temporalElement/gmd:EX_TemporalExtent/gmd:extent[//gml:TimePeriod]">
 
-			<sch:let name="beginPosition" value="gml:TimePeriod//gml:beginPosition/text()"/>
-			<sch:let name="endPosition"   value="gml:TimePeriod//gml:endPosition/text()"/>
+			<sch:let name="testBegin" value="count(gml:TimePeriod//gml:beginPosition[text() != '' or @indeterminatePosition != ''])>0"/>
```

Comment:
Il testo può essere vuoto quando l'indeterminate position è `unknown` o `now`.
Nel caso in cui sia `after` o `before` la data serve.